var/spack/repos/builtin/packages/r-lobstr/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RLobstr(RPackage):
+    """lobstr provides tools in the same vein as str(), which allow you to dig into
+    the detail of an object"""
+
+    homepage = "https://github.com/r-lib/lobstr"
+    url      = "https://github.com/r-lib/lobstr/archive/v1.1.1.tar.gz"
```

Comment:
Please use the CRAN repository for the url. Also need a `list_url`.
```suggestion
    url      = "https://cloud.r-project.org/src/contrib/lobstr_1.1.1.tar.gz"
    list_url = "https://cran.r-project.org/src/contrib/Archive/lobstr"
```